devito/operator/operator.py

```diff
@@ -624,7 +624,8 @@ def _prepare_arguments(self, autotune=None, **kwargs):
 
         # Sanity check
         for p in self.parameters:
-            p._arg_check(args, self._dspace[p], am=self._access_modes.get(p))
+            p._arg_check(args, self._dspace[p], am=self._access_modes.get(p),
+                         **kwargs)
```

Comment:
Gonna be needed in PRO soon.

I also had a change in OSS while developing this branch, but then I dropped it. I left the **kwargs because I will need it in an upcoming branch.

If you think about it, it makes a lot of sense -- there's a lot of symbolic properties we might want to check they're honored before signing off on the override; properties we aren't checking today